code/about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Textildesign</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <div class="header-text">
+      <h1>Anna Leutert</h1>
+      <h2>Handweberin, Textil- und Farbdesignerin</h2>
+    </div>
+  </header>
+
+  <nav id="navigation">
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a class="active" href="about.html">Über Mich</a></li>
+      <li><a href="aktuell.html">Aktuell</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+      <li><a href="blog.html">Tipps & Tricks</a></li>
+    </ul>
+  </nav>
+
+  <main class="">
+
+    <!--über mich / vorstellen -->
+
+    <section class="about-container">
+      <section class="text-container">
+        <div>
+          <h3>Mit viel Begeisterung weben.</h3>
+          <p>Mein ganzes Leben schon bin ich fasziniert von bunten Fäden, und was daraus gemacht werden kann. Ich fühle
+            mich wohl beim Weben, das ist meine Welt. Ich Stricke auch sehr viel. Mit bunten Fäden, alles mit bunten
+            Fäden. Und da kommt schon das nächste..... bunt. Farbe! Farbe und wie sie wirkt, und warum sie so wirkt, und
+            welches ist an dieser Stelle die Beste aller Farben. Das beschäftigt mich. Das Intressiert mich. Das ist
+            meine Welt. Das ist mein Beruf.</p>
+          <p>Ich bin Mutter von 4 tollen Töchtern. </p>
+        </div>
+        <!--Trennlinie-->
+        <hr class="solid">
+        <div>
+          <h3>Werdegang</h3>
+          <section class="tabelle">
+            <div class="line">
+              <div class="year">
+                1981 - 1984
+              </div>
+              <div>
+                Ausbildung, Handweberin EFZ, BFF Bern
+              </div>
+            </div>
+            <div class="line">
+              <div class="year">
+                1984
+              </div>
+              <div>
+                Praktikum, Leinenweberei Bern              </div>
+            </div>
+            
+
+
+          </section>
+        </div>
+      </section>
+      <div class="picture">
+        <img src="/img/portrait.JPG" alt="portrait" style="width:100%;max-width:500px;">
+      </div>
+    </section>
+
+  </main>
+
+  <footer>
+    <!-- newsletter -->
+    <section class="newsletter-container">
```

Comment:
This comment is an overall thought: GREAT naming of your tags. You can easily follow your code and understand what divs contains what content, and what they're for. Keep up the good work! 🌟